src/main/java/net/minestom/server/collision/PhysicsUtils.java

```diff
@@ -44,7 +44,7 @@ public final class PhysicsUtils {
                 physicsResult.originalDelta(), physicsResult.collisionPoints(), physicsResult.collisionShapes(), physicsResult.hasCollision(), physicsResult.res());
     }
 
-    private static @NotNull Vec updateVelocity(@NotNull Pos entityPosition, @NotNull Vec currentVelocity, @NotNull Block.Getter blockGetter, @NotNull Aerodynamics aerodynamics,
+    public static @NotNull Vec updateVelocity(@NotNull Pos entityPosition, @NotNull Vec currentVelocity, @NotNull Block.Getter blockGetter, @NotNull Aerodynamics aerodynamics,
```

Comment:
It will be nice if you can add some documentation for this `public` method

Reply:
Done in the latest comment.